src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -63,13 +63,14 @@ public TermsOfUseHelper(TermsOfUseHelperParams touParams)
             authenticationManager = touParams.AuthenticationManager;
         }
 
-        public void Execute()
+
+        public void Execute(bool preventReentrant)
```

Comment:
Can you give a default value of `true` to `preventReentrant`? Then revert all those `true` cases below, leave only the one for `false`.